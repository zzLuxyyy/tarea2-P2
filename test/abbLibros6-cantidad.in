crearABBLibrosVacio
cantidadABBLibros
crearFecha 01/05/1992
crearLibro 140003446 "Jazz" "Toni" "Morrison" "Una historia de amor y crimen en el Harlem de los años 20" 3
insertarLibroABBLibros
cantidadABBLibros
crearFecha 01/01/1971
crearLibro 204578912 "Las venas abiertas de América Latina" "Eduardo" "Galeano" "Un ensayo crítico sobre la historia de la explotación en América Latina" 7
insertarLibroABBLibros
cantidadABBLibros
crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1
insertarLibroABBLibros
cantidadABBLibros
removerLibroABBLibros 204578912
cantidadABBLibros
crearFecha 01/09/2005
crearLibro 999001122 "Nunca me abandones" "Kazuo" "Ishiguro" "Jóvenes clonados enfrentan su destino en un mundo distópico" 2
insertarLibroABBLibros
cantidadABBLibros
crearFecha 01/10/1995
crearLibro 843397030 "Ensayo sobre la ceguera" "José" "Saramago" "Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano" 2
insertarLibroABBLibros
cantidadABBLibros
liberarABBLibros
Fin
