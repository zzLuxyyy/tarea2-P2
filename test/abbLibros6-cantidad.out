1>Se ejecutó crearTABBLibrosVacio exitosamente.
2>La cantidad de libros en el árbol es 0.
3>4>5>Se insertó el libro con ISBN 140003446 al árbol.
6>La cantidad de libros en el árbol es 1.
7>8>9>Se insertó el libro con ISBN 204578912 al árbol.
10>La cantidad de libros en el árbol es 2.
11>12>13>Se insertó el libro con ISBN 316769177 al árbol.
14>La cantidad de libros en el árbol es 3.
15>El libro con ISBN 204578912 se removió del árbol.
16>La cantidad de libros en el árbol es 2.
17>18>19>Se insertó el libro con ISBN 999001122 al árbol.
20>La cantidad de libros en el árbol es 3.
21>22>23>Se insertó el libro con ISBN 843397030 al árbol.
24>La cantidad de libros en el árbol es 4.
25>26>Fin.
